.devcontainer.json

```diff
@@ -14,6 +14,10 @@
     "xaver.clang-format",
     "zxh404.vscode-proto3"
   ],
+  "mounts" : [
```

Comment:
Done. Note that as the comment says, I haven't found a good way how to do it so that it would be correct in all use cases -- if your `/dev/kvm` GID is not 107, this will break.